pkgs/by-name/ra/redbean/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  libPath = lib.makeLibraryPath [ stdenv.cc.cc  ];
+  arch =
+    if stdenv.is64bit
+    then "x86_64"
+    else "x86";
+in
+stdenv.mkDerivation rec {
+  pname = "Redbean";
+  version = "2.2";
+
+  src = fetchurl {
+    url = "https://redbean.dev/redbean-${version}.com";
+    sha256 = "db8fc7cc5a7703b7ccb830a366eb69e728fc7892fd3ecc093c089d837aa5b91b";
+  };
+
+  unpackPhase = "cp $src redbean.com";
```

Comment:
This isn't really needed? You can simply do `cp $src $out/bin/readbean.com` in your `installPhase`